app/Services/Pdf/PdfMock.php

```diff
@@ -309,10 +309,10 @@ public function getStubVariables(): array
     '$invoice.custom2' => 'custom value',
     '$invoice.custom3' => 'custom value',
     '$invoice.custom4' => 'custom value',
-    '$company.custom1' => $this->company->custom_value1,
-    '$company.custom2' => $this->company->custom_value2,
-    '$company.custom3' => $this->company->custom_value3,
-    '$company.custom4' => $this->company->custom_value4,
+    '$company.custom1' => $this->company->settings->custom_value1,
+    '$company.custom2' => $this->company->settings->custom_value2,
+    '$company.custom3' => $this->company->settings->custom_value3,
+    '$company.custom4' => $this->company->settings->custom_value4,
```

Comment:
Good pickup, this is correct.

Reply:
Thanks, I actually missed the `$contact1`-style shortcuts until now so I'll push those up in a moment.